learning/basics/task3/solutions/AC-For_loop_iteration.py

```diff
@@ -0,0 +1,21 @@
+
+prime_numbers = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97]
+
+while True:
+    try:
+        user_type = int(input("Type a number in range 1-100 -> "))
+    except ValueError:
+        print("NUMBER please!")
+    else:
+        if user_type in range(1, 100):
+            break
+        else:
+            print("This number is not within range, please enter a valid number.!")
+
+for i in prime_numbers:
+    if user_type in prime_numbers:
+        print(f"{user_type} is prime!")
+        break
+    else:
+        print(f"{user_type} is complex!")
+        break
```

Comment:
Lifehack: If you want to use `in` operator, there is no need to use `for` loop :D Another way to solve this was to use `for` then check for each element if it is equal to provided by user `for i in prime_numbers:` then `if i == user_type`, but version with `in` operator as you did is much more pythonic way to solve this
```suggestion
if user_type in prime_numbers:
    print(f"{user_type} is prime!")
else:
    print(f"{user_type} is complex!")
```